src/main/java/application/MenuController.java

```diff
@@ -137,9 +139,11 @@ public EntityDetailListResponse getDetails(@RequestBody SessionNavigationBean se
             throw new RuntimeException("Could not find case with ID " + detailSelection);
         }
 
-        EntityDetailListResponse response = new EntityDetailListResponse(entityScreen,
+        EntityDetailListResponse response = new EntityDetailListResponse(
```

Comment:
still feels like we're doing a little extra work than we need to in this route, but don't think it's anything significant